src/store/FetchData.vuex.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable max-classes-per-file */
+import { createModule } from 'vuex-class-component';
+import { AxiosInstance } from 'axios';
+
+const VuexModule = createModule({
+  strict: false,
+});
+
+interface CacheData {data: any};
+interface CacheError {error: Error};
+
+export function isCacheData(cache: CacheData | CacheError): cache is CacheData {
+  return !!(cache && (cache as CacheData).data);
+}
```

Comment:
Couldn't data be `false` or `0`? If not maybe you can change the interface to `{data: object}`